spec/index.bs

```
@@ -939,13 +1008,15 @@ dictionary IdentityProviderClientMetadata {
 </xmp>
 
 <div algorithm>
-To <dfn>select an account</dfn> given an |accountsList|, run the following steps. This returns an
-{{IdentityProviderAccount}} or failure.
-    1. Assert |accountsList|'s [=list/size=] is greater than 1.
-    1. Display an account chooser displaying the options from |accountsList|.
-    1. Let |account| be the {{IdentityProviderAccount}} of the account that the user
-        manually selects from the accounts chooser, or failure if no account is selected.
-    1. Return |account|.
+To <dfn>select an account</dfn> given |allAccountsAndProviders|, run the following steps. This
+returns an ({{IdentityProviderAccount}}, {{IdentityProviderAPIConfig}}) [=tuple=] or the [=tuple=]
+(failure, failure).
```

You are doing this so that you always return a 2-tuple? I don't think this is necessary... I'd just say "or failure"

---

It's not necessary but it was cleaner to always return a tuple here instead of maybe returning failure and then having to do checks about it in the caller.